include/arch/arm/aarch32/cortex_m/scripts/linker.ld

```diff
@@ -342,42 +377,6 @@ SECTIONS
 #endif
 
         } GROUP_LINK_IN(RAMABLE_REGION)
-
-    SECTION_DATA_PROLOGUE(_DATA_SECTION_NAME,,)
-	{
-	__data_ram_start = .;
-	*(.data)
-	*(".data.*")
-	*(".kernel.*")
-
-/* Located in generated directory. This file is populated by the
- * zephyr_linker_sources() Cmake function.
- */
-#include <snippets-rwdata.ld>
-#ifdef CONFIG_SOC_RWDATA_LD
-#include <soc-rwdata.ld>
-#endif
-
-#ifdef CONFIG_CUSTOM_RWDATA_LD
-/* Located in project source directory */
-#include <custom-rwdata.ld>
-#endif
-
-#ifdef CONFIG_CODE_DATA_RELOCATION
-#include <linker_sram_data_relocate.ld>
-#endif
-
-	} GROUP_DATA_LINK_IN(RAMABLE_REGION, ROMABLE_REGION)
-
-    __data_rom_start = LOADADDR(_DATA_SECTION_NAME);
-
-#include <linker/common-ram.ld>
-#include <linker/priv_stacks.ld>
-#include <linker/kobject.ld>
-
-#include <linker/priv_stacks-noinit.ld>
-#include <linker/cplusplus-ram.ld>
-
     __data_ram_end = .;
```

Comment:
`__data_ram_end` must be moved too.

Reply:
Sorry for lost __data_ram_end, I move it too, Thansk !